### PR TITLE
correct source command

### PR DESCRIPTION
**Pull request by nikugogoi**
fix a typo in the documentation

### CODE REVIEW

**Comment by L-Hudson** [comment]
@thaJeztah please take a look?

File: machine/install-machine.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ done
 ```
 
 Then you need to run `source
-/etc/bash_completion.d/docker-machine-prompt.bash` in your bash
+/etc/bash_completion.d docker-machine-prompt.bash` in your bash
```
Comment (thaJeztah) [inline machine/install-machine.md]: doesn't look correct to me;

```bash
source /etc/bash_completion.d foo
-bash: source: /etc/bash_completion.d: is a directory
```

can you describe the problem that you're seeing?

Comment (L-Hudson) [inline machine/install-machine.md]: Closing ticket for lack of response. If you feel this is in error, feel free to reopen. Thanks!

**Comment by L-Hudson** [comment]
closing ticket for lack of response. If you feel this is in error, feel free to reopen.

**Pull request closed without merging**